crates/backend/src/codegen.rs

```diff
@@ -904,6 +912,10 @@ impl TryToTokens for ast::Export {
                 inform(#nargs);
                 #describe_args
                 #describe_ret
+                inform(#TYPE_DESCRIBE_MAP);
+                #ret_ty_map
+                inform(#TYPE_DESCRIBE_MAP);
+                #args_ty_map
```

Comment:
I don't follow why we need two separate serializations here. The first-level cut-off of the generics is the function type, so we should just fold the data into describe_args and describe_ret instead of duplicating here.